corehq/apps/app_manager/views/releases.py

```diff
@@ -69,6 +69,7 @@ def _get_error_counts(domain, app_id, version_numbers):
 @require_deploy_apps
 def paginate_releases(request, domain, app_id):
     limit = request.GET.get('limit')
+    limit_to_released = json.loads(request.GET.get('limit_to_released'))
```

Comment:
nit: using the word `limit` here is a little confusing, since the `limit` param also exists and does something else. Maybe `released_only`?

Reply:
updated